src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,26 @@
+import { useTodos } from '../../context/TodosContext';
+import { TodoItem } from '../TodoItem';
+
+export const TodoList: React.FC = () => {
+  const { todos, filterStatus } = useTodos();
+
+  const filteredTodos = todos.filter(todo => {
+    switch (filterStatus) {
+      case 'Active':
```

Comment:
Better use Enum instead of the hardcoded string

Reply:
I Using Enum here. My switch get filterStatus as parameter and this  filterStatus is type FilterStatus = keyof typeof Status;

So if I type in my Switch case key that doesn't exist in my Status keys I will get an error.
```
export enum Status {
  All = '#/',
  Active = '#/active',
  Completed = '#/completed',
}